### ISSUE TITLE
atscm push failed: can not add null reference

### ISSUE THREAD

**Issue by FKurniawanHuisman** [atSCM/atscm#323]
Hello,

I tried to use atscm version v1.0.0-beta.39 together with Atvise v3.3.4. However, it gives an error when I try to push.

How to reproduce the error.

1. I created a new Atvise project (empty project based on the Advanced template).
2. I run `atscm init --beta` and I configure the project to use ECMAScript2015.
   Result: successful. 
   Note: After the configuration, I didn't change anything in the generated Atviseproject.babel.js
3. I execute `atscm pull --continue` to get the source code from the Atvise project.
   Result: successful.
4. I execute `atscm import`
   Result: successful
5. I tried to directly push the freshly generated source code to Atvise back using `atscm push --continue `
   Result: failed

Below are the error descriptions after trying to push two times.

> [11:23:47] Error: Script failed: An unexpected error occurred.
> Script error in 'SYSTEM.LIBRARY.ATVISE.SERVERSCRIPTS.atscm.AddReferences' @ln36,24: can not add null reference 'ns=1;s=SYSTEM.GLOBALS' [Bad]
>     at callMethod.then.result (D:\test_atscm\repository\node_modules\atscm\src\api.js:148:29)
>     at process._tickCallback (internal/process/next_tick.js:68:7)
>  - Node: SYSTEM.GLOBALS

> [11:24:07] Error: Script failed: An unexpected error occurred.
> Script error in 'SYSTEM.LIBRARY.ATVISE.SERVERSCRIPTS.atscm.AddReferences' @ln36,24: can not add null reference 'ns=1;s=AGENT.DATASOURCES' [Bad]
>     at callMethod.then.result (D:\test_atscm\repository\node_modules\atscm\src\api.js:148:29)
>     at process._tickCallback (internal/process/next_tick.js:68:7)
>  - Node: AGENT.DATASOURCES
 
Could you help me pointing out what I did wrong?

Thank you.

Kind regards,
Ferry Kurniawan
 
---

<details><summary><strong>Environment</strong></summary>

| package       |               version |
| :------------ | --------------------: |
| OS            |              Windows 7|
| atscm         |         1.0.0-beta.39 |
| atscm-cli     |                 0.8.2 | 
| atvise server |                 3.3.4 |
| node          |               10.16.3 |
| npm           |                6.11.2 |
 
</details>

<details><summary><strong>Project configuration</strong></summary>

```javascript
// Atviseproject.babel.js

import { Atviseproject } from 'atscm';

/**
 * atvise-scm configuration of test_atscm.
 */
export default class TestAtscm extends Atviseproject {

  /**
   * The atvise-server's host
   * @type {String}
   */
  static get host() {
    return 'localhost';
  }

  /**
   * The atvise-server ports to use.
   * @type {Object}
   * @property {Number} opc The OPC-UA port the atvise-server runs on.
   * @property {Number} http The HTTP port the atvise-server can be reached at.
   */
  static get port() {
    return {
      opc: 4840,
      http: 80,
    };
  }

}

```

</details>

**Comment by breussm** [comment]
Hello Mr. @FKurniawanHuisman,
we can also adjust the error with the current version.
We'll look at the topic as soon as possible and let them know.

Regards Mathias

**Comment by LukasHechenberger** [comment]
Fixed in *1.0.0-beta.40*. Run `atscm update --beta` or `npm i --save-dev atscm@beta` to update.

Thanks for reporting!
